gio/Gir.toml

```diff
@@ -307,6 +307,19 @@ status = "generate"
         name = "name"
         string_type = "os_string"
 
+[[object]]
+name = "Gio.AsyncInitable"
```

Comment:
Should also be added to the other crates in the `manual` list so that it is implemented for the types that uses it

Reply:
Which ones do you have in mind? as far as I know gtk etc does not have any async initable, no?

Reply:
Never mind, i didn't found any in gtk-rs-core/gtk3-rs/gtk4-rs